server_addon/maya/client/ayon_maya/plugins/publish/extract_ornatrix_rig.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+"""Extract Ornatrix rig."""
+
+import os
+import json
+
+from ayon_maya.api import lib
+from ayon_maya.api import plugin
+from maya import cmds
+
+
+class ExtractxRig(plugin.MayaExtractorPlugin):
+    """Extract the Ornatrix rig to a Maya Scene and write the Ornatrix rig data."""
+
+    label = "Extract Ornatrix Rig"
+    families = ["oxrig"]
+    scene_type = "ma"
+
+    def process(self, instance):
+        """Plugin entry point."""
+        cmds.loadPlugin("Ornatrix.mll", quiet=True)
+        maya_settings = instance.context.data["project_settings"]["maya"]
+        ext_mapping = {
+            item["name"]: item["value"]
+            for item in maya_settings["ext_mapping"]
+        }
+        if ext_mapping:
+            self.log.debug("Looking in settings for scene type ...")
+            # use extension mapping for first family found
+            for family in self.families:
+                try:
+                    self.scene_type = ext_mapping[family]
+                    self.log.debug(
+                        "Using {} as scene type".format(self.scene_type))
+                    break
+                except KeyError:
+                    # no preset found
+                    pass
+
+        # Define extract output file path
+        dirname = self.staging_dir(instance)
+        settings_path = os.path.join(dirname, "ornatrix.rigsettings")
+        image_search_path = instance.data["resourcesDir"]
+
+        # add textures to transfers
+        if 'transfers' not in instance.data:
+            instance.data['transfers'] = []
+
+        resources = instance.data.get("resources", [])
+        for resource in instance.data.get('resources', []):
```

Comment:
Seems like variable could be used?
```suggestion
        resources = instance.data.get("resources", [])
        for resource in resources:
```